operator/v1alpha1/types.go

```diff
@@ -35,6 +35,9 @@ type OperatorSpec struct {
 
 	// logging contains glog parameters for the component pods.  It's always a command line arg for the moment
 	Logging LoggingConfig `json:"logging,omitempty"`
+
+	// revisionHistoryLimit is the number of installer pod attempts to keep a record of (0 being unlimited)
+	RevisionHistoryLimit int `json:"revisionHistoryLimit,omitempty"`
```

Comment:
This seems way too generic.

Reply:
Also, none of the other operators are going to be able to use this struct because they have multiple image pull specs, so this is already somewhat off the rails of being truly generic.

Reply:
Yeah, @damemi I think we need this field here: https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/pkg/apis/kubeapiserver/v1alpha1/types.go#L32

This will be used only in our static pod based operators (kube-controller-manager and kube-scheduler and kube-apiserver).